x/ref/lib/vdl/codegen/json/const.go

```diff
@@ -0,0 +1,116 @@
+// Package json implements JSON generation for VDL const values.
+package json
+
+import (
+	"fmt"
+	"strconv"
+
+	"v.io/v23/vdl"
+	"v.io/x/ref/lib/vdl/codegen"
+	"v.io/x/ref/lib/vdl/vdlutil"
+	"v.io/x/ref/lib/vdl/codegen/vdlgen"
+)
+
+// Const returns a JSON representation of a value.
+//
+// TODO(razvanm): use this function to replace most (all?) of the untypedConst
+// function from v.io/x/ref/lib/vdl/codegen/javascript.
+func Const(v *vdl.Value, pkgPath string, imports codegen.Imports) string {
+	switch v.Kind() {
+	case vdl.Bool:
+		if v.Bool() {
+			return "true"
+		}
+		return "false"
+	case vdl.Byte, vdl.Uint16, vdl.Uint32:
+		return strconv.FormatUint(v.Uint(), 10)
+	case vdl.Int8, vdl.Int16, vdl.Int32:
+		return strconv.FormatInt(v.Int(), 10)
+	case vdl.Uint64:
```

Comment:
There's no advantage to using fmt.Sprintf here over strconv.Format{Uint,Int}.

Also note that javascript.untypedConst has special-cases for the 64-bit integers because they cannot be represented without loss of precision.  To be precise, JSON doesn't specify any precision limits, but typical javascript implementations use doubles (64-bit floats) to represent all numbers, which means we can represent up to +/-2^53 (or something like that) without loss.

If you're don't care about this detail for now, I'd suggest keeping the 64-bit cases separate, changing the implementation to strconv.Format{Uint,Int}, and adding comments reminding us of the loss of precision.